### PR TITLE
Update rspec syntax from should(rspec 2) to expect(rspec 3)

### PR DESCRIPTION
**Pull request by hvenables**
Vimgolf was using the old syntax of rspec. I thought i would quickly update this before getting started on extending the code coverage

### CODE REVIEW

File: spec/support/mongoid.rb [hunk 1]
```diff
@@ -36,4 +36,15 @@ def have_fields(*args)
...
+  # rspec-rails 3 will no longer automatically infer an example group's spec type
+  # from the file location. You can explicitly opt-in to the feature using this
+  # config option.
+  # To explicitly tag specs without using automatic inference, set the `:type`
+  # metadata manually:
+  #
+  #     describe ThingsController, :type => :controller do
+  #       # Equivalent to being in spec/controllers
+  #     end
+  configuration.infer_spec_type_from_file_location!
```
Comment (zoso10) [inline spec/support/mongoid.rb]: we already have this configuration set in the [spec helper](https://github.com/igrigorik/vimgolf/blob/master/spec/spec_helper.rb#L28)

**Comment by igrigorik** [comment]
LGTM, modulo @zoso10's comment. Thanks for taking this on!

File: spec/support/mongoid.rb (hunk 1 above)
Comment (hvenables) [inline spec/support/mongoid.rb]: Cool, I have amended the pull request

**Comment by igrigorik** [comment]
Hmm, odd, now Travis is failing. Could you take a quick look at what's happening there?

File: spec/support/mongoid.rb (hunk 1 above)
Comment (hvenables) [inline spec/support/mongoid.rb]: Cool, made the change. Also had to include the mongoid matchers in the spec helper

**Comment by hvenables** [comment]
Yeah needed to include the mongoid matchers in the spec helper, should pass this time

**Comment by igrigorik** [comment]
👍 

@zoso10 I'll let you hit the merge button if you're happy with this. :)

**Pull request merged**